nx.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This root level nx.json configures stuff project wide. 

We can also add nx files to individual packages to do more granular configuration